### DISCUSSION TITLE
nios_next_ip fails

### DISCUSSION THREAD

**Discussion by gcoronae** [infobloxopen/infoblox-ansible#127] [Q&A]
Hello... I thought I posted this question earlier but I don't see it.  So I  apologize if it comes up twice.  I'm trying to using this collection in our dev environment but it keeps error out at the "infoblox-client".  We are running Automation Platform 4.1.2 with python3.8 on rhel 8.5.  We installed the infoblox-client using "pip3 install infoblox-client" but we still get an error... see attachment.
![Ansible_Job_Failure](https://user-images.githubusercontent.com/42649819/175450455-d81b7bcc-ca39-4318-91b9-7667b7b4dd79.jpg)

**Comment by philipsd6** [comment]
I'm not 100% how this version of AAP handles jobs, but I believe it's using an execution environment, so you will need to ensure that the `infoblox-client` is included there -- being installed on the platform is irrelevant.
